lib/features/mailbox_creator/presentation/mailbox_creator_controller.dart

```diff
@@ -51,47 +47,35 @@ class MailboxCreatorController extends BaseController {
   @override
   void onInit() {
     super.onInit();
-    log('MailboxCreatorController::onInit():arguments: ${Get.arguments}');
-    arguments = Get.arguments;
-  }
-
-  @override
-  void onReady() {
-    super.onReady();
-    log('MailboxCreatorController::onReady():');
+    MailboxCreatorArguments? arguments = Get.arguments;
     if (arguments != null) {
-      personalMailboxTree = arguments!.personalMailboxTree;
-      defaultMailboxTree = arguments!.defaultMailboxTree;
-      teamMailboxesTre = arguments!.teamMailboxesTree;
-      accountId = arguments!.accountId;
-      _session = arguments!.session;
-      if (arguments!.selectedMailbox != null) {
-        selectedMailbox.value = arguments!.selectedMailbox;
-      }
+      personalMailboxTree.value = arguments.personalMailboxTree;
+      defaultMailboxTree.value = arguments.defaultMailboxTree;
+      selectedMailbox.value = arguments.selectedMailbox;
       _createListMailboxNameAsStringInMailboxLocation();
     }
   }
 
   @override
   void onClose() {
-    log('MailboxCreatorController::onClose():');
+    listMailboxNameAsStringExist = [];
+    _createdMailbox = false;
     nameInputFocusNode.dispose();
     nameInputController.dispose();
+    listFolderScrollController.dispose();
     super.onClose();
   }
 
   MailboxNode? _findMailboxNodeById(MailboxId mailboxId) {
-    final mailboxNode = defaultMailboxTree?.findNode((node) => node.item.id == mailboxId)
-     ?? personalMailboxTree?.findNode((node) => node.item.id == mailboxId)
-     ?? teamMailboxesTre?.findNode((node) => node.item.id == mailboxId);
```

Comment:
why you remove it? because right now, we can not create mailbox in teammailbox

Reply:
Exactly, so we don’t need to look for the mailbox in the teammailbox list.